packages/react-transform/src/index.ts

```diff
@@ -7,7 +7,7 @@ import {
 	template,
 } from "@babel/core";
 import { isModule, addNamed } from "@babel/helper-module-imports";
-import type { VisitNodeObject } from "@babel/traverse";
+import { type VisitNodeObject } from "@babel/traverse";
```

Comment:
This bumps up the minimum supported TS version considerably, same issue as #451. No real need for it either

Reply:
Biome changed it for some reason

Reply:
Hm, that sounds like a weird default. Fair enough though.